web/modules/custom/mymodule/src/Controller/FirstController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * @file 
+ * Generate markup to be displayed. Functionality in this controller is
+ * wired to Drupal in mymodule.routing.yml.
+ */
```

Comment:
I don't think this docblock is added to class files.
Please check in Drupal core to see how it is done.